proto/walletnode/register_artwork.proto

```diff
@@ -125,8 +97,10 @@ message SendSignedNFTTicketRequest {
     string key1 = 3;
     string key2 = 4;
     EncoderParameters encode_parameters = 5;
-    map<string, bytes> encode_files = 6;
+    bytes dd_fp_files = 7;
```

Comment:
should be :
    bytes dd_fp_files = **6;**
    bytes rq_files = **7;**

Reply:
ok